components/nordigen/nordigen.app.mjs

```diff
@@ -1,11 +1,61 @@
+import axios from "axios";
+
 export default {
   type: "app",
   app: "nordigen",
-  propDefinitions: {},
+  propDefinitions: {
+    country_code: {
+      type: "string",
+      label: "Enter a country code",
+      description: "ISO 3166 two-character code for the country (eg. `FR`). Full list: https://www.iso.org/obp/ui/en/#iso:pub:PUB500001:en",
+    },
+    institution_id: {
+      type: "string",
+      label: "Institution",
+      description: "Select your institution",
+      async options({ country_code }) {
+        const institutions = await this.listInstitutions(country_code); //TODO: Why isn't it working ?
```

Comment:
I'm trying to retrieve the selected country code but the value is undefined. Can't understand why.

Reply:
I'm trying to retrieve the selected country code but the value is undefined. Can't understand why. @vellames-turing